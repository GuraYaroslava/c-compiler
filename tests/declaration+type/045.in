char* p = "qwerty";

int a[5];

p + a[0];