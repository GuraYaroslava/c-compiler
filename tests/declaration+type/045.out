table 0:
`int`
`float`
`char`

table 1:
variable `p` type of pointer to `char`
variable `a` type of array 5 of `int`

tree 0:
         p
    = <
       "qwerty"

tree 1:
         p
    + <
   array: 
              a
   index: 
              0

